### DISCUSSION TITLE
Option to always show  pending transactions

### DISCUSSION THREAD

**Discussion by ridicolos** [dbeaver/dbeaver#21380] [Ideas]
I would really like a feature that would allow me to always view the `pending transactions`. You can only get to this view if you have an SQL editor open and then press `database` -> `pending transactions` under the menu item.

For example, if I rename a table via the GUI, a transaction is started in the background. But I see nothing of it in the GUI itself. This is only noticeable when I want to close DBeaver. Then I get the message that there are still open transactions. Then I first have to open a new `SQL editor`, `SQL console` or something else so that I can get to the `pending transactions` via the menu item to commit the open transaction there. If I don't open any extra window (like the editor, console or something else) the menu item won't show under the menu `database`.

In the window itself, all open transactions on all databases are displayed anyway. Why is the menu so hidden and only accessible under certain conditions?

I could imagine that the menu could simply be attached to the sidebar, e.g. like the `background jobs`, so that you can directly access all your open transactions from there.

**Comment by E1izabeth** [comment]
It sounds interesting. You could create a feature request

**Reply by ridicolos** [reply]
Thanks. I created a Feature-Request https://github.com/dbeaver/dbeaver/issues/21518
